### PR TITLE
Add behavioral guidelines in line with summary

### PR DESCRIPTION
**Pull request by eloquence**
This draws liberally from the cited codes (Tor, OpenStack, Django)
as well as FPF's organizational code (which is also based on Django's).

### CODE REVIEW

File: CoC.txt [hunk 1]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
 they feel someone has violated this Code of Conduct, or if they have questions
 or concerns.
 
+================================================================================
+Behavioral Guidelines
+================================================================================
+
+As a member of the community, we ask you to:
```
Comment (ghost) [inline CoC.txt]: I would skip this because the CoC could apply in contexts where _community_ has to be defined and the meaning of _we_ is also subject to interpretation. For instance when this CoC is used by FPF as a non-profit, the **we** is different from when this CoC is used by groups of people who have no ties with FPF.

File: CoC.txt [hunk 2]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
...
+
+- Assume good faith and good intentions. Disagreements — both social and
+  technical — happen all the time. Our diverse backgrounds give us the ability
+  to see things from many different perspectives, but they can also lead to
+  misunderstandings. It is important that we resolve disagreements
+  constructively, and that we do not jump to conclusions about each other's
+  motives.
+
+- Be welcoming, considerate, and respectful. We strive to be a community that
+  welcomes and supports people of all backgrounds and identities. This includes,
```
Comment (ghost) [inline CoC.txt:78]: What about **People of all backgrounds and identities deserve to be welcomed and supported** ?

File: CoC.txt [hunk 3]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
...
+  but is not limited to, members of any race, ethnicity, culture, national
+  origin, color, immigration status, social and economic class, educational
+  level, sex, sexual orientation, gender identity and expression, age, size,
+  family status, political belief, religion and mental and physical ability.
+
+- Be careful in the words you choose. We are a community of professionals, and
+  we conduct ourselves professionally. Exclusionary behavior is absolutely
```
Comment (ghost) [inline CoC.txt]: I suggest we skip **We are a community of professionals, and we conduct ourselves professionally.** so this CoC can also be used by people who are not professionals. Or it could be replaced by a similar sentence that does not sugest this only concerns people who are professionally working on a project.

File: CoC.txt [hunk 4]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
...
+  - Unwelcome sexual attention
+  - Advocating for, or encouraging, any of the above behavior
+
+- Listen and communicate openly and honestly. Yield the floor to
+  others, and help to make sure that everyone gets heard. In other words, try to
+  be your best self. In doing so, you contribute the health and longevity of
+  this community.
+
+As a member of the SecureDrop community, we ask you to never engage in the
+following behaviors:
```
Comment (ghost) [inline CoC.txt]: For the same reasons as above (avoiding we and community to not tie the CoC to a specific context), we could write instead **You are expected to never engage in the following behaviors:**

File: CoC.txt [hunk 5]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
...
+  - Unwelcome sexual attention
+  - Posting (or threatening to post) other people's personally identifying
+    information ("doxing")
+
+- Following the letter of this Code of Conduct while disregarding its spirit.
+  When judging whether certain behavior represents a violation of this code, we
+  will consider a violation in spirit (e.g., clearly behaving in a damaging or
+  obnoxious manner in a way not explicitly specified) to be no different from
+  any other violation of this code. That includes trolling and other forms of
+  consistently disruptive behavior.
```
Comment (ghost) [inline CoC.txt:126]: I would suggest rewording this to not use _we_. Something along the lines of **Certain behavior represents a violation of this code in spirit (e.g., clearly behaving in a damaging or obnoxious manner in a way not explicitly specified; trolling and other forms of consistently disruptive behavior) and are no different from any other violation of this code.**

**Comment by ghost** [comment]
Comments inline, thanks for getting this moving :-)

File: CoC.txt [hunk 6]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
...
+  - Personal insults
+  - Unwelcome sexual attention
+  - Advocating for, or encouraging, any of the above behavior
+
+- Listen and communicate openly and honestly. Yield the floor to
+  others, and help to make sure that everyone gets heard. In other words, try to
+  be your best self. In doing so, you contribute the health and longevity of
+  this community.
+
+As a member of the SecureDrop community, we ask you to never engage in the
```
Comment (heartsucker) [inline CoC.txt]: Can we add misgendering and deadnaming to this?

Comment (heartsucker) [inline CoC.txt]: Can we also add something about addiction and drug/substance use or lack thereof? i know a lot of people don't like when communities are like Boyz Drinkin' Clubz or whatever. Those who use/abstain shouldn't be given a hard time about it.

File: CoC.txt [hunk 7]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
...
+
+- Following the letter of this Code of Conduct while disregarding its spirit.
+  When judging whether certain behavior represents a violation of this code, we
+  will consider a violation in spirit (e.g., clearly behaving in a damaging or
+  obnoxious manner in a way not explicitly specified) to be no different from
+  any other violation of this code. That includes trolling and other forms of
+  consistently disruptive behavior.
+
 ================================================================================
 License and Attribution
```
Comment (heartsucker) [inline CoC.txt:129]: Just since I'm trying to keep up with licensing / attribution, if you took any of this directly or nearly, could you make sure to include that down here. I'd like to give credit where it's due. :)

**Review by heartsucker — COMMENTED** [review]
Thanks for keeping this moving. I've been juggling a lot and I'm glad you were able to pick up where I left off :)

A couple of minor notes I'd like to add too.

File: CoC.txt [hunk 8]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
...
+    target an individual
+  - Taking photographs, video, or audio recordings or recordings without consent
+  - Shouting
+  - Sustained disruption of talks and events
+  - Disruption of meetings
+  - Inappropriate physical contact
+  - Unwelcome sexual attention
+  - Posting (or threatening to post) other people's personally identifying
+    information ("doxing")
+
```
Comment (heartsucker) [inline CoC.txt]: I think this would be a good spot to mention general boundary violations. I know that's kinda vague, but if people need to not be talked to or not be asked any personal questions or not share food or like whatever, there may be a reason. This kinda covers one of the major cases of the "obey the spirit, not just the letter" and might make it easier to enforce the grey areas should be need to. For example.

> Part of having a diverse community withe diverse backgrounds, people you interact with will have different levels of comfort with different topics or social behaviors. Please respect people's stated boundaries, and do not attempt change these boundaries through your actions. This includes repeated requests or demands against a stated boundary.

I think there could be a better way to phrase that, but I'm not exactly an expert on this. :neutral_face:

File: CoC.txt (hunk 1 above)
Comment (eloquence) [reply CoC.txt]: I think that using "we" in some contexts has a positive, necessary, affirming quality: *We* own our commitment to these principles. *We* take it seriously. *We* are accountable for enforcing the code.  The passive voice in contexts like this is evasive.

But I understand what you're saying about it being important to identify who is described by the word. Perhaps language like "We, your fellow community members, .." could work, to state clearly that in this context, we are speaking as peers independent of our organizational affiliations.

File: CoC.txt (hunk 8 above)
Comment (eloquence) [reply CoC.txt]: I really appreciate that point, and I think "Respect people's stated personal boundaries" is pretty clear. Unpacking that further risks going more into territory where people want to rules-lawyer about it; I think we can keep it short and sweet.

**Comment by eloquence** [comment]
Thanks for the review and comments! :) I've addressed most of them and would appreciate another look.

Regarding the use of the word "we", I don't agree fully with @dachary's comments. I think it's important for us to (as a community) "own" this process, so I do think in various locations we can and should say *we*. As for the concern that it's not clear who's speaking, I would suggest adding the following language to the first occurrence of the word "We" in the preamble:

"As your fellow members of this community, we ..."

That makes it clear that it is a "We" that is encompassing us as individuals, regardless of our organizational affiliations. With that qualifier, I think the use of the word "we" in other parts of the CoC is not problematic, and in fact affirming our recognition of our shared responsibilities. Thoughts?

File: CoC.txt (hunk 7 above)
Comment (eloquence) [reply CoC.txt:129]: Thanks for flagging, added FPF and OpenStack as inspirations, as well.

File: CoC.txt [hunk 9]
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
...
+- Listen and communicate openly and honestly. Yield the floor to
+  others, and help to make sure that everyone gets heard. In other words, try to
+  be your best self. In doing so, you contribute the health and longevity of
+  this community.
+
+As a member of the SecureDrop community, you are expected to never engage in the
+following behaviors:
+
+- Intimidation, harassments, or insults.  This includes but is not limited to:
```
Comment (heartsucker) [inline CoC.txt:103]: Could we get deadnaming/misgendering here? I feel like that one is harmful and easily dismissed enough by "well I didn't know" that it should be called out.

Comment (eloquence) [reply CoC.txt:103]: Sorry, missed that on first pass. Done!
